scripts/reboot

```diff
@@ -14,6 +14,16 @@ function stop_sonic_services()
     sleep 3
 }
 
+function clear_warm_boot()
+{
+    # If reboot is requested, make sure the outstanding warm-boot is cleared
+    # So the system will come up from a cold boot.
+    WB_DIR="/host/warmboot"
+    if [[ -d ${WB_DIR} ]]; then
+        rm -f ${WB_DIR}/*.json
```

Comment:
I'd suggest to use `rm -fr ${WB_DIR}/*`

Reply:
I thought about that. The reason I did this way was because removing the db backup files will make sure the next time o.s. boots up, database won't be restored with may or may not complete files.

I particularly didn't want to remove sai-warmboot.bin because that might be needed for investigate potential SAI warmboot bugs.

But if we backup these files before reboot, then rm -rf ${WB_DIR}/* would be the right choice.

I just thought about rename the folder, but then realized that this would rename the folder at every reboot.

I am looking at if I can trap on warm-reboot failures and rename the folder. So far I only found way to execute code on all exit codes. I'll explore this approach a bit more.

Reply:
So follow your thought, the safest way is:
1. try 'config warm_reboot' to disable all the flags and save the ConfigDB
2. if config failed, remove ConfigDb json file
All the other files could be kept for troubleshooting purpose.

Reply:
We had offline discussion. For this clean up action. What we would like to achieve is:

- Do not have warm boot enabled flag persist across o.s. reboot.
- Leave any warm boot persistent store files behind for future analysis.

With that in mind, we decide to rename config_db.json as our clean up method. So all files except config_db.json would be persisted across the reboot as-is. config_db.json will be persisted with a new name.